lib/azure/armrest/virtual_machine_service.rb

```diff
@@ -20,11 +20,11 @@ def initialize(armrest_configuration, options = {})
       # as "Basic_A1", though information is included as well.
       #
       def series(location)
-        unless @@providers_hash[provider.downcase] && @@providers_hash[provider.downcase]['locations/vmSizes']
```

Comment:
The old code is still valid except the need to change keys to lower case, i.e., `location/vmsizes` because PR #116 normalized all keys to lower case.